### ISSUE TITLE
Fails to swap some tokens on main net due to 1inch token mist

### ISSUE THREAD

**Issue by huntingtonanalytics** [mxnster/dex-swapper#1]
The program fails to swap any token not listed in the 1inch token list. It would be beneficial to replace the api call to 1inch token list with https://tokenlists.org/

**Comment by mxnster** [comment]
> The program fails to swap any token not listed in the 1inch token list. It would be beneficial to replace the api call to 1inch token list with https://tokenlists.org/

Could you please provide token name?

**Comment by UnderGCrypto** [comment]
Would also love if we could somehow replace the api call to 1inch for token data with a local json or maybe https://tokenlists.org. This would allow us to use the bot on different chains on which 1inch isn't deployed (eg. linea).

**Comment by mxnster** [comment]
> Would also love if we could somehow replace the api call to 1inch for token data with a local json or maybe https://tokenlists.org. This would allow us to use the bot on different chains on which 1inch isn't deployed (eg. linea).

Yes, i have plans to improve this bot, because a lot has changed in how these dexes work since the release

**Comment by UnderGCrypto** [comment]
Great to hear it. Looking forward to it!
